pkgs/development/libraries/jsoncpp/default.nix

```
@@ -28,10 +28,16 @@ stdenv.mkDerivation rec {
     export LD_LIBRARY_PATH="`pwd`/src/lib_json''${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH"
   '';
 
-  nativeBuildInputs = [ cmake python ];
+  nativeBuildInputs = [ cmake python validatePkgConfig ];
 
-  # fix inverted sense in isAnyCharRequiredQuoting on arm. See: https://github.com/open-source-parsers/jsoncpp/pull/1120
-  patches = stdenv.lib.optionals (stdenv.isAarch64 || stdenv.isAarch32) [
+  patches = [
+    # Fix generation of pkg-config file (https://github.com/open-source-parsers/jsoncpp/pull/1199)
+    (fetchpatch {
+      url = "https://github.com/lopsided98/jsoncpp/commit/b05a21342a646a986b11c28ba6b19665756d21d2.patch";
```

We could just replace it with a project link, since it is now part of the project git repo:

```suggestion
      url = "https://github.com/open-source-parsers/jsoncpp/b05a21342a646a986b11c28ba6b19665756d21d2.patch";
```